cves/CVE-2013-0306.yml

```diff
@@ -49,7 +49,10 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  It was discovered that Django incorrectly handled a large number of forms
+  when generating formsets. An attacker could use this flaw to cause Django to
+  consume memory, resulting in a denial of service.
```

Comment:
A sentence about what fix was implemented would be useful here

Reply:
Added!